modules/oc-mirror-custom-resources-v2.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected-v2.adoc
+// * updating/updating_a_cluster/updating_disconnected_cluster/mirroring-image-repository.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="oc-mirror-IDMS-ITMS-about_{context}"]
+= Custom resources generated by v2
```

Comment:
@subhtk Is not this content duplicated with `talking about IDMS and ITMS` ?
Here: modules/oc-mirror-IDMS-ITMS-about.adoc